add-ons/che/che.addon

```diff
@@ -8,8 +8,10 @@ oc apply -f streams/che-server-streams.json -n openshift
 echo [CHE] Create the Che server Template
 oc apply -f templates/che-single-user.yml -n openshift
 
-echo [CHE] Creating mini-che project
-oc new-project mini-che --description="Eclipse Che on minishift" --as=developer
+echo [CHE] Creating mini-che project as developer
+oc login -u developer -p developer
+oc new-project mini-che --description="Eclipse Che on minishift"
+oc login -u system:admin
```

Comment:
@ibuziuk Do we need to run the rest of the command as system:admin ? I would suggest to use `--as system:admin` instead of the current approach.

Reply:
`Do we need to run the rest of the command as system:admin ?`
I guess all `che.addon` commands are run as system:admin.
`I would suggest to use --as system:admin instead of the current approach.`
Do you suggest to create a project as a system admin ? In this case the namespace will not be available for the developer user which is expected IMO